pkg/configurator/validating_webhook.go

```diff
@@ -30,13 +30,13 @@ var (
 	deserializer = codecs.UniversalDeserializer()
 
 	// boolFields are the fields in osm-config that take in a boolean
-	boolFields = []string{"egress", "enable_debug_server", "permissive_traffic_policy_mode", "prometheus_scraping", "tracing_enable", "use_https_ingress"}
+	boolFields = []string{"egress", "enable_debug_server", "permissive_traffic_policy_mode", "prometheus_scraping", "tracing_enable", "use_https_ingress", "privileged_init_container"}
```

Comment:
Can we use is/use/enable_privileged_init_container / something in that vein? I know it makes the var name long but for the sake of consistency with the other boolean vars eg: `enable_debug_server`, `tracing_enable`, `use_https_ingress` and the way it's named in all the other files